include/mgba/internal/gba/hardware-extensions.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2021 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#ifndef HARDWARE_EXTENSIONS_H
+#define HARDWARE_EXTENSIONS_H
```

Comment:
`GBA_EXTENSIONS_H`

Reply:
I'm guessing I also should rename hardware-extensions.c to extensions.c. And change the preffix from "GBAHardwareExtensions" to "GBAExtensions", right?

Reply:
Probably a good idea, and move it into extra/ since it's not core functionality.

Reply:
Fixed.